pkg/server/test/write.go

```diff
@@ -832,6 +860,54 @@ var writeCommandTests = []writeCommandTest{
 			},
 		),
 	},
+	{
+		_name: "ObsoleteModel1.0",
```

Comment:
We're adopting test names with underscores in our other test code throughout the project. Do you mind if we do the same here? `obsolete_model_1_0` or something like that.